api/taxonomies/serializers.py

```diff
@@ -25,6 +25,13 @@ class TaxonomySerializer(JSONAPISerializer):
     text = ser.CharField(max_length=200)
     parents = JSONAPIListField(child=TaxonomyField())
     child_count = ser.IntegerField()
+    children = ser.SerializerMethodField('get_child_num')
+
+    def get_child_num(self, obj):
```

Comment:
If you just make this method `get_children` then you don't have to set the method on the serializer method field.